pkg/block/s3/adapter.go

```diff
@@ -411,6 +411,7 @@ func (a *Adapter) GetPreSignedURL(ctx context.Context, obj block.ObjectPointer,
 
 	log := a.log(ctx).WithFields(logging.Fields{
 		"operation":  "GetPreSignedURL",
+		"sid":        obj.StorageID, // TODO (gilo): is "sid" a good enough name?
```

Comment:
Definitely not good enough :) 
Let's be explicit (we don't have to be stingy in our logs)

Reply:
The main reason I haven't done this is that StorageNamespace is logged as `namespace` and not as `storage_namespace` 😖